### PR TITLE
Audit plugin phase 1

### PR DESCRIPTION
**Pull request by antonyagustine**
**CS-41076**

### CODE REVIEW

File: packages/contentstack/package.json [hunk 1]
```diff
@@ -37,11 +38,10 @@
     "@contentstack/cli-migration": "~1.3.12",
     "@contentstack/cli-utilities": "~1.5.2",
     "@contentstack/management": "~1.10.0",
-    "@contentstack/cli-cm-branches": "~1.0.12",
```
Comment (shafeeqd959) [inline packages/contentstack/package.json:40]: can we add this back ?

File: packages/contentstack-audit/bin/dev [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
```
Comment (shafeeqd959) [inline packages/contentstack-audit/bin/dev:1]: can we use the latest method for the dev ?

File: packages/contentstack-audit/.editorconfig [hunk 3]
```diff
@@ -0,0 +1,11 @@
+root = true
```
Comment (shafeeqd959) [inline packages/contentstack-audit/.editorconfig:1]: can we use prettier config instead of editor ?

File: packages/contentstack-audit/bin/dev (hunk 2 above)
Comment (antonyagustine) [reply packages/contentstack-audit/bin/dev:1]: What do you mean by `latest method for the dev`? Can you rephrase it.?

File: packages/contentstack-audit/.editorconfig (hunk 3 above)
Comment (antonyagustine) [reply packages/contentstack-audit/.editorconfig:1]: Sure, I can remove it and commit. However, I need not push one for prettier we have it in our CLI global config itself.

File: packages/contentstack/package.json (hunk 1 above)
Comment (antonyagustine) [reply packages/contentstack/package.json:40]: Sure

File: packages/contentstack-audit/.editorconfig (hunk 3 above)
Comment (shafeeqd959) [reply packages/contentstack-audit/.editorconfig:1]: that works

File: packages/contentstack-audit/bin/dev (hunk 2 above)
Comment (shafeeqd959) [reply packages/contentstack-audit/bin/dev:1]: #!/usr/bin/env node
require('dotenv').config({ path: './.env' });
(async () => {
  const { execute } = require('@contentstack/cli-utilities');
  await execute({ type: 'cjs', development: true, dir: __dirname });
})();

we could use this instead

**Review by netrajpatel — DISMISSED** [review]

**Review by aman19K — APPROVED** [review]
